activerecord/lib/active_record/connection_adapters/postgresql/oid/type_map_initializer.rb

```diff
@@ -37,11 +37,18 @@ def query_conditions_for_initial_load
             known_type_names = @store.keys.map { |n| "'#{n}'" }
             known_type_types = %w('r' 'e' 'd')
             <<-SQL % [known_type_names.join(", "), known_type_types.join(", ")]
+              LEFT join pg_type as composite_children on t.typelem=composite_children.oid
```

Comment:
Can you upcase SQL keywords (`join`, `as`, `on`) to keep the existed convention?

```
% git grep -n -i ' join ' lib/active_record/connection_adapters/postgresql
lib/active_record/connection_adapters/postgresql/explain_pretty_printer.rb:12:        #    Nested Loop Left Join  (cost=0.00..37.24 rows=8 width=0)
lib/active_record/connection_adapters/postgresql/explain_pretty_printer.rb:13:        #      Join Filter: (posts.user_id = users.id)
lib/active_record/connection_adapters/postgresql/schema_statements.rb:74:            INNER JOIN pg_index d ON t.oid = d.indrelid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:75:            INNER JOIN pg_class i ON d.indexrelid = i.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:76:            LEFT JOIN pg_namespace n ON n.oid = i.relnamespace
lib/active_record/connection_adapters/postgresql/schema_statements.rb:92:            INNER JOIN pg_index d ON t.oid = d.indrelid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:93:            INNER JOIN pg_class i ON d.indexrelid = i.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:94:            LEFT JOIN pg_namespace n ON n.oid = i.relnamespace
lib/active_record/connection_adapters/postgresql/schema_statements.rb:164:                LEFT JOIN pg_namespace n ON n.oid = c.relnamespace
lib/active_record/connection_adapters/postgresql/schema_statements.rb:335:              JOIN pg_attribute   attr ON (t.oid = attrelid)
lib/active_record/connection_adapters/postgresql/schema_statements.rb:336:              JOIN pg_attrdef     def  ON (adrelid = attrelid AND adnum = attnum)
lib/active_record/connection_adapters/postgresql/schema_statements.rb:337:              JOIN pg_constraint  cons ON (conrelid = adrelid AND adnum = conkey[1])
lib/active_record/connection_adapters/postgresql/schema_statements.rb:338:              JOIN pg_namespace   nsp  ON (t.relnamespace = nsp.oid)
lib/active_record/connection_adapters/postgresql/schema_statements.rb:364:              JOIN pg_attribute a
lib/active_record/connection_adapters/postgresql/schema_statements.rb:508:            JOIN pg_class t1 ON c.conrelid = t1.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:509:            JOIN pg_class t2 ON c.confrelid = t2.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:510:            JOIN pg_attribute a1 ON a1.attnum = c.conkey[1] AND a1.attrelid = t1.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:511:            JOIN pg_attribute a2 ON a2.attnum = c.confkey[1] AND a2.attrelid = t2.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:512:            JOIN pg_namespace t3 ON c.connamespace = t3.oid
lib/active_record/connection_adapters/postgresql/schema_statements.rb:760:            sql = "SELECT c.relname FROM pg_class c LEFT JOIN pg_namespace n ON n.oid = c.relnamespace".dup
```

And also, since our code base expects space around operators (this check couldn't work in here docs through), so can you add spaces around `=`?

https://github.com/rails/rails/blob/018ff582a5aaa58d4bce7d634ecae80d107f8304/.rubocop.yml#L107-L109